src/services/metrics-recorder.ts

```diff
@@ -116,14 +116,22 @@ export class MetricsRecorder {
         fallbackTag = ' FALLBACK'
       }
 
+      // Save raw methods for the errors db
+      const rawMethods = method
+
+      // Reduce multi-method calls for metrics/logging purposes
+      if (method && method.split(',').length > 1) {
```

Comment:
can you add a quick test for this. otherwise I think it looks good.

Reply:
Added in 99d371bd1813cb10c960aa7ecdfabb76c41b7164